fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBStoreTimer.java

```diff
@@ -610,6 +611,8 @@ public enum Counts implements Count {
         PLAN_DAM("number of dam plans", false),
         /** The number of synthetic record type plans. */
         PLAN_SYNTHETIC_TYPE("number of synthetic record types plans", false),
+        /** The number of plans that include a {@link RecordQueryRecursiveUnionPlan}. */
```

Comment:
Didn't you want to add other events, too?

Reply:
Not in this PR since it is already large, but yes
https://github.com/FoundationDB/fdb-record-layer/issues/3020